src/main/java/jp/co/soramitsu/sora/json/Saltifier.java

```diff
@@ -0,0 +1,66 @@
+package jp.co.soramitsu.sora.json;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+@AllArgsConstructor
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class Saltifier {
+
+  ObjectMapper mapper;
+  SaltGenerator generator;
+  String valueKey;
+  String saltKey;
+
+  public Saltifier(ObjectMapper mapper, SaltGenerator generator) {
+    this(mapper, generator, "v", "s");
```

Comment:
better to have `v` and `s` changed to explicit values `value`, `salt`

Reply:
shorter names reduce storage requirements for mobile devices and sora backend